MC_Setup.cmd

```diff
@@ -247,6 +252,7 @@ put #var MC_IMBUE ROD
 put #var MC_IMBUE.MANA 50
 put #var MC_FOCUS.WAND wood wand
 put #var MC_Enchanting_NOWO 0
+put #var MC_PREFERRED.ROOM.ARTIF NULL

```

Comment:
For where you want to go to do your enchanting work. I created this unique variable because there may be mana requirements on the room you choose, unlike the global work outside room